tests/test_http_ext_auth.py

```diff
@@ -1229,7 +1215,9 @@ async def test_http_auth_ext_azure_authorize_01(self):
             self.assertEqual(url.scheme, "https")
             self.assertEqual(url.hostname, "login.microsoftonline.com")
             self.assertEqual(url.path, "/common/oauth2/v2.0/authorize")
-            self.assertEqual(qs.get("scope"), ["openid profile email"])
+            self.assertEqual(
+                qs.get("scope"), ["openid profile email offline_access"]
```

Comment:
Note this scope has the additional `offline_access` scope we configured in the setup for the Azure provider.